sections/06-vibrant-bodies.md

```diff
@@ -0,0 +1,201 @@
+
+Whatever happens to the turntable and the analogue vinyl record, James Brown, after all, is never broken by a digital sampler's manipulation of him, nor is his (or the sampler's) essence somehow lost or activated by the process. Transformation is general and ubiquitous when it is digital. Although it may seem that it is digital media's reproducibility that affords such zero cost transformation without breakage, reproducibility, in itself, is just another form of breakdown. Every software programmer knows the impossible tangle of duplication that is the result of the copy/paste procedure. So perhaps it is not the digital's infinite reproducibility that allows it to avoid destruction in the act of recreation, but its capture within a logic of signals and loops: the repetition, modulation, and filtration of materials into something new. "It is this act, replaying, that marks the real rupture in the mode or method of production" (Mudede).
+
+Echoing this idea of "replaying" and drawing it back to the theme of community, Stephen Voyce imagines a more general mode of what he calls "open source poetics," which is defined by "a decentralized and nonproprietary model of shared cultural codes, networks of dissemination, and collaborative authorship" (407). His view of open communities and their relation to art practice focusses on appropriative, readymade, remix, and "uncreative" strategies, which call into question the role of singular authorship and shift creative emphasis instead towards how poetry contributes to a "poetic commons" of textual references and transformations (408). While, as discussed above, I think it is a mistake to focus only on the citational or appropriative modes of open communities, I nonetheless want to explore Voyce's idea of a commons in which works of art serve, via quotation and systematic transformation, as the "raw materials," or signals, for new works.
+
+Notably, Voyce argues that, underneath the agonistic and oppositional tropes of modernism, open community tactics have been an ongoing feature of avant-garde art movements such as Dada and Fluxus. For Voyce, this method of inter-authorial materiality suggests that avant-garde artists have an ongoing role and responsibility "to create and fortify public domains of open source knowledge, to challenge excessive restrictions placed on language and information, to bring forth marginalized knowledges from a position of inaccessibility to the public at large, and to produce and share artistic tactics and works that challenge intellectual property. That which is at stake is nothing less than open accessibility to culture" (427-8).
```

Comment:
I think various incarnations of fan work (fanfiction, etc) are another interesting example of "open community tactics" from the sphere of popular (rather than artistic - let's be Cartesian dualists about it) culture.

See the [Organization for Transformative Works](http://beta.transformativeworks.org/) for more than you are probably interested in hearing about all this. :)